### PR TITLE
Added visualization feature to demo utils

### PR DESCRIPTION
**Pull request by divyashreepathihalli**
Demo of loading VOC data and visualizing using bounding box.
vis_demo

### CODE REVIEW

File: examples/layers/preprocessing/bounding_box/demo_utils.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+from keras_cv import bounding_box
+
+
+def resize(image, label, img_size=(224, 224), num_classes=10):
+    image = tf.image.resize(image, img_size)
+    label = tf.one_hot(label, num_classes)
+    return {"images": image, "labels": label}
+
+
+def load_oxford_dataset(
```
Comment (LukeWood) [inline examples/layers/preprocessing/bounding_box/demo_utils.py:29]: you can remove the oxford functions from the bounding box utils

File: examples/layers/preprocessing/bounding_box/demo_utils.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            "objects": {
+                "bbox": tf.cast(-1, tf.float32),
+                "is_difficult": None,
+                "is_truncated": None,
+                "label": None,
+                "pose": None,
+            },
+        },
+    )
+    return next(iter(dataset.take(1)))
```
Comment (LukeWood) [inline examples/layers/preprocessing/bounding_box/demo_utils.py:81]: any reason we're taking next(iter()) here?  We should be showing off how to use the KPLs on the entire dataset, not just a batch.

Let's return the dataset

Comment (LukeWood) [inline examples/layers/preprocessing/bounding_box/demo_utils.py:81]: I recommend mapping the dataset before returning in the following way:

```
def package_to_dict(inputs):
    outputs = {"images": ds["image"], "bounding_boxes": ds["objects"]["bbox"]}
    return outputs
```

**Review by LukeWood — CHANGES_REQUESTED** [review]
Minor changes

also, you can't use `examples.layers.preprocessing.classification.demo_utils` if people want to run the scripts from any directory.   then they have to run from the top level which is a bit annoying to users, lets revert `import examples.layers.preprocessing.classification.demo_utils as demo_utils` => demo_utils

File: examples/layers/preprocessing/bounding_box/demo_utils.py (hunk 1 above)
Comment (divyashreepathihalli) [reply examples/layers/preprocessing/bounding_box/demo_utils.py]: Done!

File: examples/layers/preprocessing/bounding_box/demo_utils.py (hunk 2 above)
Comment (divyashreepathihalli) [reply examples/layers/preprocessing/bounding_box/demo_utils.py]: Done!

Comment (divyashreepathihalli) [reply examples/layers/preprocessing/bounding_box/demo_utils.py]: Done!

**Comment by LukeWood** [comment]
Thank you Divya!

**Pull request merged**